file_organisation.qmd

```diff
@@ -545,6 +549,26 @@ questions:
 ```
 ## `R/custom_R_code.R`
 
-XXX 
+The  [`austraits.build`](https://github.com/traitecoevo/austraits.build/) compilation contains an extra folder, `R` containing a file `custom_R_code.R`. This file documents any custom functions used in the compilation, called as part of the [`custom_R_code` section](#dataset) of metadata files. 
+
+It includes functions to:
+
+- Replace duplicate trait values with NA's
+
+- Convert various month formats into a string of 12 NY's (to document flowering, fruiting, recruitment times)
+
+- Move specific categorical trait values to a second trait
+
+For instance, there are many datasets where a species-level trait measurement is repeated across many rows of data but should only be incorporated into the dataset a single time:
+
+```
+custom_R_code: `
+  data %>%
+    mutate(
+      across(c(`plant_growth_form`, `leaf_shape`), replace_duplicates_with_NA)
```

Comment:
Make clear that this function is part of AusTraits, not traits.build

Reply:
This section is explicitly about the custom_R_code.R file in austraits.build - header is `## R/custom_R_code.R`